# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/classicube/font-path.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

diff '--color=auto' -ruN ClassiCube.og/src/Platform_Posix.c ClassiCube/src/Platform_Posix.c
--- ClassiCube.og/src/Platform_Posix.c	2024-04-26 15:24:02.070839180 +0000
+++ ClassiCube/src/Platform_Posix.c	2024-04-26 15:02:23.399718327 +0000
@@ -544,6 +544,7 @@
 	};
 #else
 	static const cc_string dirs[] = {
+		String_FromConst("/usr/X11/share/fonts"),
 		String_FromConst("/usr/share/fonts"),
 		String_FromConst("/usr/local/share/fonts")
 	};
